testsuite/classlibrary/TestUnitTest.sc

```diff
@@ -1,13 +1,34 @@
 TestUnitTest : UnitTest {
 
-	var someVar;
+	var someVar, setUpHappened = false;
+	classvar setUpHappened = false;
+	classvar tearDownHappened = false;
```

Comment:
could you please not use the same name for the classvar and var? it's confusing

Reply:
yes, of course.